### ISSUE TITLE
Android Manifest error

### ISSUE THREAD

**Issue by khoamoc** [myflashlab/GPS-ANE#14]
Hi, 
I just bought your GPS ANE yesterday ( because Adobe native Geolocation class NOT working with Android anymore)
iOS version is OK ( i only include gps.ane ) but Android build having error, it takes me the whole day to debug it but invain. Is there any conflict with others ANE from different sources? ( i currently use ANEs from Distriqt and Milkman games ,too)
It seems that the error comes from gps.ane.

Here's the error and my currently using [ANEs

![untitled](https://cloud.githubusercontent.com/assets/13786375/15273595/7b87b95c-1a53-11e6-88f2-34190e919ab7.png)

![untitled-1](https://cloud.githubusercontent.com/assets/13786375/15273597/81da010c-1a53-11e6-8763-6b06ead3209d.gif)

**Comment by myflashlab** [comment]
> It seems that the error comes from gps.ane manifest xml

Not exactly, if you are using other ANEs in your project, you must make sure they don't have conflicting packages. one of the most common conflicting packages are the *Google Games Services" package. our GPS ANE use that package and other ANEs from other developers may have that too.

We have made sure that our [dependency ANEs](https://github.com/myflashlab/common-dependencies-ANE) are global so all ANE providers could use them for free to make sure their ANEs are not conflicting with ours. Anyhow, just to make sure, please run your project with our ANE only and then add other ANEs one by one to see which one is conflicting.

> Can you tell me the "name" of the activity tag? 

There is no activity tag to be added to your manifest. we have embedded the required functionality inside the ANE. you just need to add the following services to your manifest:

```xml
<!-- required for GPS -->
<service android:name="io.nlopez.smartlocation.geocoding.providers.AndroidGeocodingProvider$AndroidGeocodingService" android:exported="false" />
<service android:name="io.nlopez.smartlocation.activity.providers.ActivityGooglePlayServicesProvider$ActivityRecognitionService" android:exported="false" />
<service android:name="io.nlopez.smartlocation.geofencing.providers.GeofencingGooglePlayServicesProvider$GeofencingService" android:exported="false" />
<meta-data android:name="com.google.android.gms.version" android:value="@integer/google_play_services_version" />

```
[check here](https://github.com/myflashlab/GPS-ANE#air-xml-manifest) for more information about how you should set the manifest

other possible problems is that maybe you are trying to run the ANE in shared mode? please don't, [just try the debug or the captive build](https://github.com/myflashlab/GPS-ANE#requirements).

Another probable issue might be your Air SDK version. Please upgrade to the latest version because we are using the latest Google Play Services version. Air SDK 20 or higher should be ok.

And finally make sure to watch these videos. https://www.youtube.com/watch?v=Oubsb_3F3ec&list=PL_mmSjScdnxnSDTMYb1iDX4LemhIJrt1O

**Comment by myflashlab** [comment]
Please update this ticket if the problem is solved. Thanks.
